CONTRIBUTING.md

```diff
@@ -21,6 +19,8 @@ During the normal course of development on the VSCode extension you'll want to r
 pnpm dev
 ```
 
+> Note: If you haven't ran `pnpm build` or `pnpm dev` already, you may see some errors related to some files being missing. This is normal on a first run, and you can safely ignore these errors.
```

Comment:
```suggestion
> [!NOTE] 
> If you haven't ran `pnpm build` or `pnpm dev` already, you may see some errors related to some files being missing. This is normal on a first run, and you can safely ignore these errors.
```

This uses the new GitHub markup https://github.com/orgs/community/discussions/16925

Reply:
Oh yeah forgot about that, good catch.